README.md

```diff
@@ -73,6 +73,14 @@ versions of the compiler.
 [rustup.rs](https://www.rustup.rs/) may be used to manage
 installations of multiple versions of rust on single machine.
 
+The currently supported Rust version is nightly-2016-05-24.  To install
+it with rustup use the following:
```

Comment:
It's probably useful to also provide a tip to look at .travis.yml and see which version is being used there (just to cover the case where this info isn't up-to-date for some reason.